lntest/harness_assertion.go

```diff
@@ -388,7 +388,7 @@ func (h *HarnessTest) AssertOutputScriptClass(tx *btcutil.Tx,
 
 	pkScript, err := txscript.ParsePkScript(txOut.PkScript)
 	require.NoError(h, err)
-	require.Equal(h, pkScript.Class(), scriptClass)
+	require.Equal(h, scriptClass, pkScript.Class())
```

Comment:
this seems unnecessary, is there something I'm missing?

Reply:
The require library assumes the first argument after the test struct is the "expected" value while the next is the "actual" value. If those are swapped, the generated test failure message might be confusing. So it's not strictly necessary but a nice fix.